component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -359,16 +361,18 @@ private void processValidUserToken(AuthenticationContext context, AuthenticatedU
     }
 
     /**
-     * Checks the backup codes for email otp.
+     * Check whether the entered code matches with a backup code.
      *
-     * @param context           AuthenticationContext.
-     * @param userToken         EmailOTP token.
-     * @param authenticatedUser AuthenticatedUser
-     * @throws AuthenticationFailedException
+     * @param context The AuthenticationContext.
+     * @param userToken The userToken.
+     * @param authenticatedUser The authenticatedUser.
```

Comment:
Shall we fix formatting in the method comment?

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/84/commits/04b1581704622d8e7af97178e36c0406370a951d